Makefile

```diff
@@ -212,11 +222,26 @@ test: generate manifests test-utils images/dev-env/.dockerbuilt ## Run unit test
 
 ##@ Advanced building targets
 
-test-utils: timer multithread_tracee pkg/time/fakeclock/fake_clock_gettime.o pkg/time/fakeclock/fake_gettimeofday.o
+# Build chaosctl locally
+local/chaosctl:
+	$(GO) build -ldflags '$(LDFLAGS)' -o bin/chaosctl ./cmd/chaosctl/main.go
```

Comment:
it seems that the original recipe `chaosctl` was already been build locally, so there is no difference between `make local/chaosctl` and the old `make chaosctl`.

Or is there other things I missed?

Reply:
Yes, it is. Only the name has changed.

> After discussing with @STRRL, we decided not to change the old locally built targets.